src/main/kotlin/lotto/core/WinningNumbers.kt

```diff
@@ -2,14 +2,17 @@ package lotto.core
 
 import lotto.core.constant.LottoConstants
 
-data class WinningNumbers(val winningNumbers: List<Int>) {
+class WinningNumbers(numbers: List<Int>, private val bonusNumber: Int) {
+    private val lotto: Lotto = Lotto(numbers)
```

Comment:
실제 프로퍼티를 주 생성자에 정의하고 다른 타입을 생성자 파라미터로 받고싶다면 부생성자를 쓰는 것이 더 자연스러운 흐름인 것 같아요. 🤔

```suggestion
class WinningNumbers(
    private val lotto: Lotto = Lotto(numbers),
    private val bonusNumber: Int,
) {
    
    constructor(numbers: List<Int>, private val bonusNumber: Int): this(Lotto(numbers), bonusNumber)
```

Reply:
lotto 프로퍼티를 외부에 노출하지 않기 위해 우회로 사용한 방법인데, 의견 주신대로 부생성자로 받아서 상관없을거 같습니다.
부생성자로 받도록 수정하겠습니다.